distributed/scheduler.py

```diff
@@ -291,14 +291,18 @@ def clean(self):
         return ws
 
     def __repr__(self):
-        return "<Worker %r, memory: %d, processing: %d>" % (
+        return "<Worker %r, name: %s, memory: %d, processing: %d>" % (
```

Comment:
Should we include this if name is None or the same as the address?

Reply:
Agreed. I can change that (tomorrow)

Reply:
I got second thoughts during the implementation of the proposal and therefore left it out for now:

* In order to add an `if` for the optional name, we would need to provide the scheduler implementation with the knowledge that worker defaults its name to its address. This would break encapsulation.
* On the UI we also always show the name field even if it is identical to the worker address.

If you insist I am happy make the name optional though.

Reply:
OK, I'm happy to accept the push back.